charts/seed-bootstrap/charts/elastic-kibana-curator/templates/elasticsearch/statefulset.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: elasticsearch-logging
   namespace: {{ .Release.Namespace }}
   labels:
-    garden.sapcloud.io/role: logging
+    gardener.cloud/role: logging
```

Comment:
Please, add the old label key `garden.sapcloud.io`, too.
It is needed for the deletion of the logging resources.